build/ci/golangci.yml

```diff
@@ -44,9 +44,12 @@ linters-settings:
 
 # https://golangci-lint.run/usage/linters
 linters:
+  disable-all: true
+  # Some linters are not compatible with go 1.18, so they are temporary disabled.
+  # See https://github.com/golangci/golangci-lint/issues/2649
   enable:
     - asciicheck
-    - bodyclose
+    #- bodyclose # Not compatible with go 1.18, temporary disable
```

Comment:
Niektore `linters` nefunguju s `go 1.18`,
Tu je tabulka: https://github.com/golangci/golangci-lint/issues/2649

Reply:
![image](https://user-images.githubusercontent.com/19371734/170653873-05fb39a1-a780-48b6-b2ac-19c576672057.png)

Reply:
Cakam, ze behom par mesiacov budu fixnute aj dalsie a vratime to spat.